docs/src/main/sphinx/release/release-441.md

```diff
@@ -0,0 +1,24 @@
+# Release 441 (13 Mar 2024)
+
+## General
+
+* Fix incorrect results of window aggregations when any input data includes 
+  `NaN` or infinity. ({issue}`20946`)
+
+## Cassandra connector
+
+* Fix incorrect results when a query contains predicates on clustering columns. ({issue}`20963`)
```

Comment:
clustering columns ?? is that a term people understand?

Reply:
I'm not sure how else I would word it

Reply:
Yes, that's a concept in Cassandra